### PR TITLE
Update TestGrid to use load balancer

### LINKED ISSUE #5392: DNS REQUEST: testgrid
### Type of DNS update:

Update

### Domain being modified:

testgrid.k8s.io and testgrid.kubernetes.io

### Existing DNS Record:

```yaml
testgrid:
  - type: A
    values:
    - 216.239.32.21
    - 216.239.34.21
    - 216.239.36.21
    - 216.239.38.21
  - type: AAAA
    values:
    - 2001:4860:4802:32::15
    - 2001:4860:4802:34::15
    - 2001:4860:4802:36::15
    - 2001:4860:4802:38::15
```

### New DNS Record:

```yaml
testgrid:
  - type: A
    values:
    - 34.120.51.46
```

### Reason for update:

Updating the TestGrid site to use a load balancer instead of the default App Engine access.

### PR DESCRIPTION
**Pull request by michelle192837**
Fixes #5392

### CODE REVIEW

**Comment by michelle192837** [comment]
/hold

Fixing commits 🙃

File: dns/zone-configs/k8s.io._0_base.yaml [hunk 1]
```diff
@@ -326,16 +326,7 @@ submit-queue:
...
-    - 216.239.34.21
-    - 216.239.36.21
-    - 216.239.38.21
-  - type: AAAA
-    values:
-    - 2001:4860:4802:32::15
-    - 2001:4860:4802:34::15
-    - 2001:4860:4802:36::15
-    - 2001:4860:4802:38::15
+    - 34.120.51.46
```
Comment (BenTheElder) [inline dns/zone-configs/k8s.io._0_base.yaml:329]: no IPv6/AAAA record at all this time?

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/approve
/hold

File: dns/zone-configs/k8s.io._0_base.yaml (hunk 1 above)
Comment (michelle192837) [reply dns/zone-configs/k8s.io._0_base.yaml:329]: Updated, thanks!

**Comment by michelle192837** [comment]
I'll unhold tomorrow when I'm around to pay attention to this (will be out in the morning Pacific Time).

**Review by BenTheElder — COMMENTED** [review]
/lgtm
/hold cancel

**Comment by BenTheElder** [comment]
/hold
oops just saw that last comment, job is failing anyhow

File: dns/zone-configs/k8s.io._0_base.yaml [hunk 2]
```diff
@@ -326,16 +326,10 @@ submit-queue:
...
-    - 216.239.36.21
-    - 216.239.38.21
+    - 34.120.51.46
   - type: AAAA
     values:
-    - 2001:4860:4802:32::15
-    - 2001:4860:4802:34::15
-    - 2001:4860:4802:36::15
-    - 2001:4860:4802:38::15
+    - 2600:1901:0:dc01::
```
Comment (BenTheElder) [inline dns/zone-configs/k8s.io._0_base.yaml:332]: this needs to be quoted, yaml parses this like `foo:` (a key `2600:1901:0:dc01:` with key/value delimeter `:` and value `nulll`)

**Comment by aojea** [comment]
❤️

File: dns/zone-configs/k8s.io._0_base.yaml (hunk 2 above)
Comment (michelle192837) [reply dns/zone-configs/k8s.io._0_base.yaml]: Ah yup, thanks for the tip!

**Comment by michelle192837** [comment]
/hold cancel

Though it'll need re-LGTM

**Pull request merged**